README.md

```diff
@@ -1,11 +1,28 @@
-## Partnership Engineering Integration Testing Framework
+## Minimal Artifactory Functional Testing Framework
 
-The purpose of this framework is to test different Artifactory configurations deployed by various Partnership Engineering solutions.
-It can be used in the CI-CD process and in a single manual test run.
-NOTE: the test purpose is to test if deployment was correct and JFrog Deployment functioning as expected. 
-The test will delete all existing repositories and create a list of new repos. Do not run on live instance. 
+This test framework has been repurposed from the partner-integration-tests used internally in JFrog by the Partnership Engineering team.
 
-Test suites include Artifactory Pro, JCR and Xray tests as well as some Data analytics tests.    
+**NOTE:** the test purpose is to test if deployment was correct and JFrog Deployment is functioning as expected and not for running  on a Production live JFrog Deployment. 
+You can use this test framework **"as is"** and  it is not supported by JFrog.
+
+It has the following customizations to the original partner-integration-tests :
+a) It is upgraded to run with :
+
+```
+❯ gradle --version
+
+Gradle 7.4.2
+Kotlin:       1.5.31
+Groovy:       3.0.9
+Ant:          Apache Ant(TM) version 1.10.11 compiled on July 10 2021
+JVM:          18 (Oracle Corporation 18+36-2087)
+```
+
+b) It does not delete any existing repositories that are already in artifactory. Instead it creates new local, remote and virtual repos ( name prefixed with “appbdd-” ) and after the test is done it deletes only these appbdd-* repos.
+
+c) I disabled the setBaseURLTest() in src/test/groovy/tests/HealthCheckTest.groovy by changing the group name from "common" to "common-notneeded"
```

Comment:
`setBaseURLTest()` is not only the test, it allows to run some of the Xray tests.

Reply:
I like your idea to add a condition to not change  the BaseUrl if it was already set